src/slider/slider.js

```diff
@@ -30,6 +30,7 @@
     this.element_ = element;
     // Browser feature detection.
     this.isIE_ = window.navigator.msPointerEnabled;
+    this.isEdge_ = !this.isIE_ && !!window.StyleMedia;
```

Comment:
I think we need to detect the error itself if possible and refactor the detection to be just for that. So instead of this `isIE_` and `isEdge_` stuff that is hacky and eventually could be misleading, we just do `hasXBug_` or something along those lines. And that is a boolean based on the detection of the error at hand.

Without more details on the error itself, which should have been included in the PR description, I can't really look into possible ways to guide you along detecting it.